repository/r.json

```diff
@@ -1443,6 +1443,17 @@
 				}
 			]
 		},
+		{
+			"name": "Restructured Text Improved",
```

Comment:
The original was named "RestructuredText Improved", without a space here. (I know because I have it installed still.) To allow older users to receive updates, I suggest re-using the old name here.

Reply:
Indeed - well spotted. Fixed.